pages/Documents/ConversationalAI/BotAnalytics/transcripts.md

```diff
@@ -32,9 +32,9 @@ For an explanation of each status, see [here](bot-analytics-key-terms-concepts.h
 
 You can use the **MACS** view to perform a targeted review of transcripts based on MACS data, such as the MACS score. For example, you might want to review the transcripts of conversations that had MACS 1 (i.e., a low conversation quality score). Or, you might want to review the transcripts of conversations where the bot didn't understand the consumer's question.
 
-<img style="width:800px" src="img/ConvoBuilder/ba_transcripts_macs.png" alt="">
+<img style="width:800px" src="img/ConvoBuilder/ba_transcripts_macs.png" alt="Accessing transcripts for review by way of a conversation list that's filtered by MACS criteria">
 
-To learn more about MACS, see [here](bot-analytics-macs.html).
+[Learn more about MACS](bot-analytics-macs.html).
```

Comment:
(Optional: For entire-sentence links, consider including punctuation in the link.)